doc/md/language-manual.md

```diff
@@ -1754,6 +1756,53 @@ Otherwise, `r1` and `r2` are values `v1` and `v2` and the expression returns the
 
 For equality and inequality, the meaning of `v1 <relop> v2` depends on the compile-time, static choice of `T` (not the run-time types of `v1` and `v2`, which, due to subtyping, may be more precise).
 
+### Pipe operators and placeholder expressions
+
+The pipe expression `<exp1> |> <exp2>` is syntactic sugar for a `let` binding to a reserved
+placeholder identifier, `p`, referenced by the placeholder expression `_`:
+
+``` bnf
```

Comment:
why `bnf`?

Reply:
That's what I use elsewhere for syntactic defs

Reply:
Rewrote the section a bit.